esmvaltool/references/esacci-soilmoisture.bibtex

```diff
@@ -1,13 +1,115 @@
 @article{esacci-soilmoisture,
-	doi = {10.5194/hess-15-425-2011},
-	url = {https://doi.org/10.5194%2Fhess-15-425-2011},
-	year = 2011,
-	month = {feb},
-	publisher = {Copernicus {GmbH}},
-	volume = {15},
-	number = {2},
-	pages = {425--436},
-	author = {Y. Y. Liu and R. M. Parinussa and W. A. Dorigo and R. A. M. De Jeu and W. Wagner and A. I. J. M. van Dijk and M. F. McCabe and J. P. Evans},
-	title = {Developing an improved soil moisture dataset by blending passive and active microwave satellite-based retrievals},
-	journal = {Hydrology and Earth System Sciences}
+    doi = {10.5194/essd-11-717-2019},
+    title = {Evolution of the {ESA} {CCI} Soil Moisture climate data records
+               and their underlying merging methodology},
+    author = {Gruber, Alexander and Scanlon, Tracy and van der Schalie, Robin
+               and Wagner, Wolfgang and Dorigo, Wouter},
+    abstract = {The European Space Agency's Climate Change Initiative
+               for Soil Moisture (ESA CCI SM) merging algorithm generates
+               consistent quality-controlled long-term (1978--2018) climate
+               data records for soil moisture, which serves thousands of
+               scientists and data users worldwide. It harmonises and merges
+               soil moisture retrievals from multiple satellites into (i) an
+               active-microwave-based-only product, (ii) a
+               passive-microwave-based-only product and (iii) a combined
+               active--passive product, which are sampled to daily global
+               images on a 0.25∘ regular grid. Since its first release in 2012
+               the algorithm has undergone substantial improvements which have
+               so far not been thoroughly reported in the scientific
+               literature. This paper fills this gap by reviewing and
+               discussing the science behind the three major ESA CCI SM merging
+               algorithms, versions 2
+               (https://doi.org/10.5285/3729b3fbbb434930bf65d82f9b00111c;
+               Wagner et al., 2018), 3
+               (https://doi.org/10.5285/b810601740bd4848b0d7965e6d83d26c;
+               Dorigo et al., 2018) and 4
+               (https://doi.org/10.5285/dce27a397eaf47e797050c220972ca0e;
+               Dorigo et al., 2019), and provides an outlook on the expected
+               improvements planned for the next algorithm, version 5.},
+    journal   = {Earth Syst. Sci. Data},
+    publisher = {Copernicus GmbH},
+    volume    =  {11},
+    number    =  {2},
+    pages     = {717--739},
+    month     =  {may},
+    year      =  2019
 }
+
+@article{esacci-soilmoisture,
```

Comment:
I am not entirely if you can specify multiple references like that. Did you check the output data if that works?

Reply:
In the output data it's including only the first doi in the reference metadata field indeed

Reply:
I see, then it probably makes sense to only include one reference in there (the most relevant one?)

Reply:
Thanks, indeed, I leave it as it is then as the first reference in the list seems the most relevant one, in case we want to include another one we just need to change the reference order in the file